addons/account/models/chart_template.py

```diff
@@ -745,45 +755,36 @@ def execute(self):
         return {}
 
     @api.multi
-    def _create_bank_journals_from_o2m(self, company, acc_template_ref):
-        '''
-        This function creates bank journals and its accounts for each line encoded in the field bank_account_ids of the
-        wizard.
-
-        :param company_id: the id of the company for which the wizard is running.
-        :param acc_template_ref: the dictionary containing the mapping between the ids of account templates and the ids
-            of the accounts that have been generated from them.
-        :return: True
+    def _create_bank_journals_from_o2m(self, company):
```

Comment:
Remove this method and field, no need to create bank accounts from the wizard. 
People will create their bank accounts from the Settings menu.